Server/MirObjects/MapObject.cs

```diff
@@ -399,6 +399,71 @@ public virtual void BroadcastInfo()
         public abstract int Attacked(PlayerObject attacker, int damage, DefenceType type = DefenceType.ACAgility, bool damageWeapon = true);
         public abstract int Attacked(MonsterObject attacker, int damage, DefenceType type = DefenceType.ACAgility);
 
+        public virtual void GetArmour(DefenceType type, MapObject attacker, out bool hit, ref int armour)
```

Comment:
Since this method is a GetArmour method - it should really be returning the armour value set - instead of having it as a ref parameter

Reply:
All should be good now.